### PR TITLE
Haproxy attrs

### PR DESCRIPTION
**Pull request by jhesketh**
Add support for downinter and tune rise/fall params.

### CODE REVIEW

File: chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb [hunk 1]
```diff
@@ -99,8 +99,9 @@ listen  admin-stats <%= node[:haproxy][:stats][:enabled] ? node[:haproxy][:stats
...
         inter = " inter #{server[:inter] || 2000}"
         fastinter = server[:fastinter] ? " fastinter #{server[:fastinter]}" : ""
-        rise = " rise #{server[:rise] || 2}"
-        fall = " fall #{server[:fall] || 5}"
+        downinter = server[:downinter] ? " downinter #{server[:downinter]}" : ""
+        rise = " rise #{server[:rise] || 5}"
+        fall = " fall #{server[:fall] || 2}"
         backup = server[:backup] ? " backup" : ""
       %>
         server <%= server[:name] %> <%= server[:address] %>:<%= server[:port] %> check<%= ssl %><%= inter %><%= fastinter %><%= rise %><%= fall %><%= backup %>
```
Comment (jsuchome) [inline chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb]: Shouldn't you add `downinter` to `server` line?

File: chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb [hunk 2]
```diff
@@ -99,8 +99,9 @@ listen  admin-stats <%= node[:haproxy][:stats][:enabled] ? node[:haproxy][:stats
         ssl = content[:use_ssl] ? " check-ssl verify none" : ""
         inter = " inter #{server[:inter] || 2000}"
         fastinter = server[:fastinter] ? " fastinter #{server[:fastinter]}" : ""
-        rise = " rise #{server[:rise] || 2}"
-        fall = " fall #{server[:fall] || 5}"
+        downinter = server[:downinter] ? " downinter #{server[:downinter]}" : ""
+        rise = " rise #{server[:rise] || 5}"
+        fall = " fall #{server[:fall] || 2}"
```
Comment (vuntz) [inline chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb]: Can you elaborate on the rise/fall change: the commit message is only describing the change and doesn't explain why we should do the change. I get the fall part, but the rise part is a bit unclear to me.

File: chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb [hunk 3]
```diff
@@ -99,6 +99,7 @@ listen  admin-stats <%= node[:haproxy][:stats][:enabled] ? node[:haproxy][:stats
         ssl = content[:use_ssl] ? " check-ssl verify none" : ""
         inter = " inter #{server[:inter] || 2000}"
         fastinter = server[:fastinter] ? " fastinter #{server[:fastinter]}" : ""
+        downinter = server[:downinter] ? " downinter #{server[:downinter]}" : ""
```
Comment (aspiers) [inline chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb]: @jsuchome is right - this new variable isn't actually getting used by anything :)

File: chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb (hunk 1 above)
Comment (jhesketh) [reply chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb]: Ah yes, I missed that. Thanks.

File: chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb (hunk 2 above)
Comment (jhesketh) [reply chef/cookbooks/haproxy/templates/default/haproxy.cfg.erb]: I'll expand in the commit as well. It is somewhat arbitrary, but basically my thinking is that we should be quick to change to another healthy host and slow to consider a returning host healthy.

**Comment by jsuchome** [comment]
@aspiers do you still request a change?

**Pull request merged**